plugins/ngrok/credentials.go

```diff
@@ -12,14 +13,17 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
+const ngrokAuthToken = "Authtoken"
```

Comment:
Let's move this into the `fieldname` package, even if it's just being used by one plugin.

Reply:
Done, thanks!